doc/sfp-cmis/cmis.md

```diff
@@ -0,0 +1,308 @@
+# Feature Name
+QSFPDD CMIS support for SONiC
+
+# High Level Design Document
+#### Rev 0.1 (Draft)
+
+# Table of Contents
+  * [List of Tables](#list-of-tables)
+  * [Revision](#revision)
+  * [About This Manual](#about-this-manual)
+  * [Scope](#scope)
+  * [Abbreviation](#abbreviation)
+  * [References](#references)
+  * [Requirement Overview](#requirement-overview)
+  * [Functionality](#functionality)
+  * [Design](#design)
+  * [Testing](#testing)
+
+# List of Tables
+  * [Table 1: Definitions](#table-1-definitions)
+  * [Table 2: References](#table-2-references)
+
+# Revision
+| Rev |     Date    |       Author        | Change Description                |
+|:---:|:-----------:|:-------------------:|-----------------------------------|
+| 0.1 | 09/27/2021  | Dante (Kuo-Jung) Su | Initial version                   |
+
+# About this Manual
+This document provides general information about the QSFPDD CMIS support for SONiC.
+
+# Scope
+This document describes functional behavior of the QSFPDD CMIS support for SONiC.
+
+# Abbreviation
+
+# Table 1: Definitions
+| **Term**       | **Definition**                                   |
+| -------------- | ------------------------------------------------ |
+| pmon           | Platform Monitoring Service                      |
+| xcvr           | Transceiver                                      |
+| xcvrd          | Transceiver Daemon                               |
+| CMIS           | QSFPDD Common Management Interface Specification |
+| QSFPDD         | Quad Small Form Factor Pluggable Double Density  |
+
+# References
+
+# Table 2 References
+
+| **Document**                                            | **Location**  |
+|---------------------------------------------------------|---------------|
+| QSFPDD Common Management Interface Specification (CMIS) | [CMIS5p0.pdf](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiL0pT5gqHzAhWWYisKHWI3CNoQFnoECCIQAQ&url=http%3A%2F%2Fwww.qsfp-dd.com%2Fwp-content%2Fuploads%2F2021%2F05%2FCMIS5p0.pdf&usg=AOvVaw3vUjsBDOeUWoYaMn3KDG_w) |
+
+# Requirement Overview
+
+This document describes functional behavior of the QSFPDD CMIS support in SONiC.
+
+The QSFPDD Common Management Interface Specification (CMIS) version 4.0 was finalized
```

Comment:
Document above references the CMIS 5.0 spec, this section is referencing CMIS 4.0. There are minimal differences, but which is being used here? Will CMIS 3.x support be included?

Reply:
CMIS 3 is not be supported, only CMIS 4 and 5 will be supported.
And thanks for the comments, the terms of 'version 4' will be removed.